app/controllers/comments_controller.rb

```diff
@@ -8,6 +8,7 @@ class CommentsController < ApplicationController
 
   def create
     if @comment.save
+      @comment.reload
```

Comment:
This is not required, since `CommentCreatedJob` isn't dependent on the saved object, and will fetch the object on perform.